src/document.cpp

```diff
@@ -620,6 +638,39 @@ rvoe<NoReturnValue> PdfDocument::create_catalog() {
   /Pages {} 0 R
 )",
                    pages_object);
+
+    if(!page_labels.empty()) {
+        buf += R"(  /PageLabels
+    << /Nums [
+)";
+        for(const auto &page_label : page_labels) {
+            std::format_to(app, R"(      {} <<)", page_label.start_page);
+            if(page_label.style) {
+                std::format_to(app,
+                               R"(
+        /S {})",
```

Comment:
This works the opposite of all other formatters. They have the endline character in the string that ends the line, not at the beginning of the next line.

Changing this also means that all the format strings fit in a single line so they don't need the `R`s.